pkg/controller/test/context_builder.go

```diff
@@ -51,7 +52,7 @@ import (
 func init() {
 	logs.InitLogs(nil)
 	_ = flag.Set("alsologtostderr", fmt.Sprintf("%t", true))
```

Comment:
same question for this flag

Reply:
it is still present and default is false https://github.com/kubernetes/klog/blob/d7fc50528570f8276de46570f916682257821e22/klog.go#L407-L408